src/NzbDrone.Core/Localization/Core/en.json

```
@@ -83,6 +83,7 @@
   "Automatic": "Automatic",
   "AutomaticAdd": "Automatic Add",
   "AutomaticSearch": "Automatic Search",
+  "AutomaticUpdatesDisabledDocker": "Automatic updates are disabled when using the Docker update mechanism. You will need to update the container image outside of Sonarr.",
```

In an effort to make this re-usable:
```suggestion
  "AutomaticUpdatesDisabledDocker": "Automatic updates are disabled when using the Docker update mechanism. You will need to update the container image outside of {appName}.",
```

Then pass `{ appName: 'Sonarr' }` into `translate`